react/features/toolbox/defaultToolbarButtons.js

```diff
@@ -153,12 +154,32 @@ export default {
         id: 'toolbar_button_dial_out',
         onClick() {
             JitsiMeetJS.analytics.sendEvent('toolbar.sip.clicked');
-            APP.store.dispatch(openDialOutDialog());
+
+            return openDialOutDialog();
         },
         tooltipKey: 'dialOut.dialOut'
     },
 
     /**
+     * The descriptor of the device selection toolbar button.
+     */
+    deviceselection: {
```

Comment:
We agreed offline that it's better to have this new button because the whole logic is pretty tight to the device selection only for now.